frontend/Cargo.toml

```diff
@@ -1,18 +1,22 @@
 [package]
-name = "trunk-template"
+name = "flunk_website"
 version = "0.1.0"
 edition = "2021"
-description = "Template for starting a Yew project using Trunk"
+description = "Frontend for Flunk Website"
 readme = "README.md"
-repository = "https://github.com/yewstack/yew-trunk-minimal-template"
+repository = "https://github.com/LukeBrandon/fluke"
```

Comment:
Not sure if this should be /frontend

Reply:
The repository is what is here, if we wanted to do submodules we could but that's just more work for now.